packages/@angular/cli/models/webpack-configs/production.ts

```diff
@@ -103,7 +103,13 @@ export function getProdConfig(wco: WebpackConfigOptions) {
     }));
   }
 
-  const uglifyCompressOptions: any = {};
+  const uglifyCompressOptions: any = {
+      // Disabled because of an issue with Uglify breaking seemingly valid code:
```

Comment:
That statement is inaccurate. See https://github.com/mishoo/UglifyJS2/issues/2520#issuecomment-347342508

Reply:
I'm not too proficient in english, but it's written "an issue with Uglify", and not "an issue in Uglify". It's two different things, no ?

Reply:
It's a mapbox-gl issue.